src/main/java/com/logpresso/scanner/Log4j2Scanner.java

```diff
@@ -677,7 +677,7 @@ private boolean isExcluded(String path) {
 	}
 
 	private boolean isVulnerable(int major, int minor, int patch) {
-		return major == 2 && (minor < 14 || (minor == 14 && patch <= 1));
+		return major == 2 && (minor < 15 || (minor == 15 && patch <= 1));
```

Comment:
If 2.15.x is vulnerable, it should be `return major == 2 && minor < 16`

Reply:
Yes ;-)